documentation/add/07-sqw_redesign.md

```diff
@@ -141,29 +141,59 @@ The `Axis` classes describes image axes
+
+The cut algorithm takes existing **sqw** object containing existing **projection** and **axes_block** classes. retrieves target **projection** and **axes_block** classes from the input binning parameters of the cut, and calculates *npix*, *signal* and *error* from the pixel information, present in the source **sqw** object or from  *npix*, *signal* and *error* of the source object if the pixel information is not present in the source object.
+
+The **axes_block** class contains three methods, necessary to implement the cut:
+
+| method | Description | Notes |
+|-----|---------|---|
+| *get_axes_scales* | returns sufficiently small 4D hyper-cube to fit any cell of the **axes_block** grid regardless of any transformation, defined by the target projection (see below) | ^1 |
+| *get_bin_nodes* | accepts the hypercube, produced by *get_axes_scales* and generates the grid which covers all range, described by the current **axes_block** class | |
+| *bin_pixels* | takes the coordinates of the pixels, expressed in the **access_block** coordinate system and, optionally, signal and error for these pixels and calculates  *npix*, *signal* and *error* from these pixels to the **axes_block** cells |  |
 
-The `ProjectionManager` class will manage all projection operations.
 
-![Projection Class Overview](../diagrams/projection.png)
+The particular projection classes are the children of the abstract **aProjection** class defining three abstract **aProjection** methods, which should be defined by child classes.
+These methods are:
 
-Responsible for all image projections - this includes symmetrization and simpler data operations:
+| method | Description | Notes |
+|-----|---------|---|
+| *transform_pix_to_img* | Transforms the points with the coordinates in the Crystal Cartesian coordinate system into the Image cordinate system | For **ortho_proj** class image coordinate system would be hkl, for **spher_proj**:  *Q,theta,phi,dE* |
+| *transform_img_to_pix* | Transform the points with the coordinate in the Image coordinate system into Crystal Cartesian coordinate system. | Operation, opposite to *transform_pix_to_img* | 
+| *get_axes_block* | returns the particular **axes_block** class, specific for the coordinate system, defined by the specific projection class | |
 
-- Crystal Cartesian to h,k,l,
-- arbitrary rotation,
-- arbitrary offset,
-- plane cuts
-- spherical cuts
-- cylindrical cuts
+Other important methods, necessary for the analysis of different cuts from an sqw object, and implemented using these transformation can be defined using these three abstract methods. These methods together with their short descriptions provided in the table:
+
+| method | Description | Notes |
+|-----|---------|---|
+| *from_cur_to_targ_coord* | Transforms the pixels of the current image coordinate system into other coordinate system the requested cut is presented in. | |
```

Comment:
Unclear what is transformed. I would guess that the individual pixels have unchanged signal and error but their positions are changed into new ones specified by the new coordinate system. Is this correct? Are pixels also moved around between bins which have also changed positions. A more detailed description like "transforms the spatial coordinates of the pixels" or somesuch.

Reply:
hopefully clarified